internal/ingress/annotations/authtls/main.go

```diff
@@ -80,6 +81,12 @@ var authTLSAnnotations = parser.Annotation{
 			Risk:          parser.AnnotationRiskLow,
 			Documentation: `This annotation defines if the received certificates should be passed or not to the upstream server in the header "ssl-client-cert"`,
 		},
+		annotationAuthTLSPassCertToUpstreamHeader: {
+			Validator:     parser.ValidateNull,
+			Scope:         parser.AnnotationScopeLocation,
+			Risk:          parser.AnnotationRiskLow,
```

Comment:
I don't think this is Low, as you are allowing users to:
* Add arbitrary string
* Change the name of a security header

Reply:
Sure, I don't know the implications of the risk levels. What do you recommend?

Reply:
IMO we can set it to high level